imap_processing/tests/ultra/unit/test_ultra_l1c_pset_bins.py

```diff
@@ -99,8 +110,7 @@ def test_get_background_rates():
     assert background_rates.shape == hp.nside2npix(128)
 
 
-@pytest.mark.external_test_data
```

Comment:
This is still needed on all of these tests to tell it that it requires external data. This will make it only download in the one runner still.